pkg/exchange/okex/exchange.go

```diff
@@ -363,3 +373,44 @@ func (e *Exchange) QueryOrder(ctx context.Context, q types.OrderQuery) (*types.O
 
 	return toGlobalOrder(order)
 }
+
+// Query order trades can query trades in last 3 months.
+func (e *Exchange) QueryOrderTrades(ctx context.Context, q types.OrderQuery) ([]types.Trade, error) {
+	if len(q.ClientOrderID) != 0 {
+		log.Warn("!!!OKEX EXCHANGE API NOTICE!!! Okex does not support searching for trades using OrderClientId.")
+	}
+
+	var req *okexapi.GetTransactionHistoriesRequest
```

Comment:
why not `e.client.NewGetTransactionHistoriesRequest()` and assign to the req at line 383?